File: tests/fixtures/language_adapter_symbols/Sample.kt
// HAND-DERIVED: standard Kotlin syntax written from language knowledge, not from any adapter regex.
package com.example.inventory

data class Item(val name: String, val quantity: Int)

class Warehouse {
    private val items = mutableListOf<Item>()

    fun addItem(item: Item) {
        items.add(item)
    }

    fun totalQuantity(): Int = items.sumOf { it.quantity }
}

interface Describable {
    fun describe(): String
}

File: tests/fixtures/language_adapter_symbols/sample.dart
// HAND-DERIVED: standard Dart syntax written from language knowledge, not from any adapter regex.
class Item {
  final String name;
  final int quantity;

  Item(this.name, this.quantity);
}

class Warehouse {
  final List<Item> items = [];

  void addItem(Item item) {
    items.add(item);
  }
}

abstract class Describable {
  String describe();
}
